lms/services/blackboard_api/_schemas.py

```diff
@@ -55,3 +55,19 @@ class Meta:
     @post_load
     def post_load(self, data, **_kwargs):  # pylint:disable=no-self-use
         return data["results"]
+
+
+class BlackboardListGroupSetGroups(RequestsResponseSchema):
+    class GroupSchema(Schema):
+        class Meta:
+            unknown = EXCLUDE
+
+        id = fields.Str(required=True)
+        name = fields.Str(required=True)
+        groupSetId = fields.Str(required=False, allow_none=True)
```

Comment:
Allow for groups without groupset in the responses.

Not required for this endpoint (group/set/{...}/groups`) but required while getting groups for students.